earthpy/plot.py

```diff
@@ -323,12 +323,17 @@ def plot_rgb(
     # Then plot. Define ax if it's default to none
     if ax is None:
         fig, ax = plt.subplots(figsize=figsize)
+        ax.imshow(rgb_bands, extent=extent)
+        ax.set_title(title)
+        ax.set(xticks=[], yticks=[])
+        # Multipanel won't work if plt.show is called prior to second plot def
+        # Thus we either have redundant code or DON"T call plt.show
+        plt.show()
```

Comment:
@mbjoseph this is the problem area. when you call plt.show() too early it populates all of the panels they end up with no data in them. so this is a workaround. 

conversely we remove plt.show() and just call this part

```
ax.imshow(rgb_bands, extent=extent)
        ax.set_title(title)
        ax.set(xticks=[], yticks=[])
```
once

Reply:
UPDATE: i figured out a cleaner way to implement this!!